inc/switchtec/mrpc.h

```diff
@@ -60,8 +60,10 @@ enum mrpc_cmd {
 	MRPC_MULTI_CFG = 29,
 	MRPC_SES = 30,
 	MRPC_RD_FLASH = 31,
+	MRPC_GAS_READ = 41,
 	MRPC_ECHO = 65,
 	MRPC_GET_PAX_ID = 129,
+	MRPC_GAS_WRITE = 135,  // CHECK ME
```

Comment:
Please check and remove comment before merging.

There's also another check me comment in the same file that should be resolved and removed.

Reply:
Firmware decided to use current values in the most recent release and update the values (to final ones, currently the firmware is still under development) in the next release. So I think we can keep the comment here till firmware change it to final ones.

Reply:
In that case, I'd say don't bother merging it until we have the proper values in place.